src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.DuplicateUserException;
+import core.basesyntax.exception.InvalidAgeException;
+import core.basesyntax.exception.InvalidLoginException;
+import core.basesyntax.exception.InvalidPasswordException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService = new RegistrationServiceImpl();
+    private static final String LOGIN = "PERSON";
+    private static final String PASSWORD = "11111111";
+    private static final String SHORT_PASSWORD = "1111";
```

Comment:
```suggestion
    private static final String SHORT_PASSWORD = "11111";
```
for testing, it is better to take marginally unacceptable input data, in this case, 5 symbols (check another testing data)